packages/flutter_tools/lib/src/artifacts.dart

```diff
@@ -17,6 +17,8 @@ import 'globals.dart' as globals;
 enum Artifact {
   /// The tool which compiles a dart kernel file into native code.
   genSnapshot,
+  /// Hannes might know what call this.
```

Comment:
was this a TODO to check with Hannes?

Reply:
I am unsure if libFlutterEngine is the correct/best name for this , since it might be named differently on different platforms , at the time of writing I thought that Hannes might have a better suggestion for a name.

Reply:
Noted.

Reply:
Done.